_config.yml

```diff
@@ -28,5 +28,3 @@ google_analytics_ua: UA-48605964-19
 
 sass:
   sass_dir: assets/_scss
-  style: :compressed
```

Comment:
@juliaelman Sure! Can you show what you were thinking about adding in a variable for production and development here? The only thing I could imagine is a separate config for dev and production, but you mentioned there's another way.